File: libgdx_drag_race/core/src/com/nopalsoft/dragracer/MainStreet.java
package com.nopalsoft.dragracer;

import com.badlogic.gdx.Game;
import com.badlogic.gdx.graphics.g2d.SpriteBatch;
import com.badlogic.gdx.scenes.scene2d.Stage;
import com.badlogic.gdx.utils.viewport.StretchViewport;
import com.nopalsoft.dragracer.screens.MainMenuScreen;
import com.nopalsoft.dragracer.screens.Screens;

public class MainStreet extends Game {

    public MainStreet() {

    }

    public Stage stage;
    public SpriteBatch batch;

    @Override
    public void create() {
        stage = new Stage(new StretchViewport(Screens.SCREEN_WIDTH,
                Screens.SCREEN_HEIGHT));

        batch = new SpriteBatch();
        Assets.load();

        setScreen(new MainMenuScreen(this));
    }
}

File: libgdx_drag_race/core/src/com/nopalsoft/dragracer/MainStreet.kt
package com.nopalsoft.dragracer

import com.badlogic.gdx.Game
import com.badlogic.gdx.graphics.g2d.SpriteBatch
import com.badlogic.gdx.scenes.scene2d.Stage
import com.badlogic.gdx.utils.viewport.StretchViewport
import com.nopalsoft.dragracer.screens.BaseScreen
import com.nopalsoft.dragracer.screens.MainMenuScreen

class MainStreet : Game() {

    lateinit var stage: Stage
    lateinit var batcher: SpriteBatch


    override fun create() {

        stage =
            Stage(StretchViewport(BaseScreen.SCREEN_WIDTH.toFloat(), BaseScreen.SCREEN_HEIGHT.toFloat()))
        batcher = SpriteBatch()

        Assets.load()


        setScreen(MainMenuScreen(this))
    }
}